config.toml

```diff
@@ -7,7 +7,7 @@ enableRobotsTXT = true
 
 disableKinds = ["taxonomy", "taxonomyTerm"]
 
-ignoreFiles = [ "^OWNERS$", "README.md", "^node_modules$", "content/en/docs/doc-contributor-tools" ]
+ignoreFiles = [ "^OWNERS$", "README*.md", "^node_modules$", "content/en/docs/doc-contributor-tools" ]
```

Comment:
Does this regex accurately exclude examples like `README-ko.md`? 

@zparnold

Reply:
`^OWNERS$` seems to copy the OWNERS file in the subdirectory. I think `OWNERS$` is correct.

Reply:
@cstoku Thanks--it's the addition of [language-specific READMEs](https://github.com/kubernetes/website/pull/10485/files#r223518697) (for example, `README-ja.md`) for which I want to verify the exclusion with `"README*.md"`.

Reply:
@zparnold ☝️ PTAL

Reply:
I think you want to tighten this regex. You want to match things that:

- start with `README`
- end with `.md`
- Don't have `-[a-z][a-z]`

So: `^README\.md` gets exactly `README.md` without any others.

If you wanted to match only the language-specific ones: `README[-]+[a-z]*\.md` That looks for things with 1 or more hyphen and then 2 or more letters before the `.md`

Reply:
@mistyhacks Thanks for the review! ✨

Reply:
@zacharysarah Use `\\` instead of `\` in a toml file, like `"README[-]+[a-z]*\\.md"`.